features/bootstrap/FeatureContext.php

```diff
@@ -60,13 +60,13 @@ public function __construct(ManagerRegistry $doctrine, Request $request)
     }
 
     /**
-     * Sets the default Accept HTTP header to the JSON-LD mime type.
+     * Sets the default Accept HTTP header to null (workaround to artificially remove it).
      *
      * @BeforeScenario
      */
     public function acceptJsonLd()
     {
-        $this->request->setHttpHeader('Accept', 'application/ld+json');
+        $this->request->setHttpHeader('Accept', null);
```

Comment:
It was my initial goal but it doesn't look possible to unset the header using Behat/Mink.
IMO it's better to have it unset by default and explicitly set it in the scenario.